src/profiled-aot/dotnet.aotprofile.txt

```diff
@@ -2087,34 +2313,50 @@ Methods:
-	void System.ProbabilisticMap:Initialize (uint*,System.ReadOnlySpan`1<char>)
-	void System.ProbabilisticMap:SetCharBit (uint*,byte)
-	void System.Random:.cctor ()
-	void System.Random:.ctor ()
-	void System.Random:.ctor (bool)
-	void System.Random/ThreadSafeRandom:.ctor ()
+	void System.Private.CoreLib.WorkerThread/<>c:<.cctor>b__13_0 ()
 	void System.Random/XoshiroImpl:.ctor ()
-	void System.ReadOnlySpan`1<System.Object>:.ctor (object[])
-	void System.ReadOnlySpan`1<System.String>:.ctor (string&,int)
 	void System.Reflection.Assembly:.cctor ()
+	void System.Reflection.AssemblyName:.ctor (string)
+	void System.Reflection.AssemblyName:FillName (Mono.MonoAssemblyName*,string,bool,bool,bool)
+	void System.Reflection.AssemblyNameFormatter:AppendQuoted (System.Text.ValueStringBuilder&,string)
+	void System.Reflection.AssemblyNameParser:.ctor (System.ReadOnlySpan`1<char>)
+	void System.Reflection.AssemblyNameParser:RecordNewSeenOrThrow (System.Reflection.AssemblyNameParser/AttributeKind&,System.Reflection.AssemblyNameParser/AttributeKind)
+	void System.Reflection.AssemblyNameParser/AssemblyNameParts:.ctor (string,System.Version,string,System.Reflection.AssemblyNameFlags,byte[])
 	void System.Reflection.ConstructorInfo:.cctor ()
+	void System.Reflection.ConstructorInvoker:.ctor (System.Reflection.RuntimeConstructorInfo)
 	void System.Reflection.CustomAttribute:.cctor ()
+	void System.Reflection.Emit.DynamicMethod:.cctor ()
+	void System.Reflection.Emit.DynamicMethod:.ctor (string,System.Type,System.Type[],System.Reflection.Module,bool)
+	void System.Reflection.Emit.DynamicMethod:create_dynamic_method (System.Reflection.Emit.DynamicMethod,string,System.Reflection.MethodAttributes,System.Reflection.CallingConventions)
+	void System.Reflection.Emit.DynamicMethod:CreateDynMethod ()
+	void System.Reflection.Emit.DynamicMethod:Init (string,System.Reflection.MethodAttributes,System.Reflection.CallingConventions,System.Type,System.Type[],System.Type,System.Reflection.Module,bool,bool)
+	void System.Reflection.Emit.ILGenerator:.ctor (System.Reflection.Module,System.Reflection.Emit.ITokenGenerator,int)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode,int)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode,System.Reflection.ConstructorInfo)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode,System.Reflection.FieldInfo)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode,System.Reflection.MethodInfo)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode,System.Type)
+	void System.Reflection.Emit.ILGenerator:Emit (System.Reflection.Emit.OpCode)
+	void System.Reflection.Emit.ILGenerator:emit_int (int)
+	void System.Reflection.Emit.ILGenerator:label_fixup (System.Reflection.MethodBase)
+	void System.Reflection.Emit.ILGenerator:ll_emit (System.Reflection.Emit.OpCode)
+	void System.Reflection.Emit.ILGenerator:make_room (int)
```

Comment:
Wait something is using System.Reflection.Emit now?!? Need to investigate what...

Reply:
This is coming from `System.Reflection.ConstructorInfo.Invoke()`.

Will investigate if we can use the feature flag: https://github.com/dotnet/runtime/pull/80246